### DISCUSSION TITLE
StandICT.eu open call

### DISCUSSION THREAD

**Discussion by Ainali** [standard-for-public-code/standard-for-public-code#1043] [General]
I am thinking about applying to the [StandICT.eu](https://www.standict.eu/standicteu-2026-6th-open-call) to be able to some maintenance on the repository. Two questions:

1. Do I have the support from the community to take this on?
2. What would be the prioritized tasks in your opinion?

For 2, I am thinking resolving the problem with the website theme and revamping the translations are high priority, but I am also open to other ideas or order of priority.

**Comment by ericherman** [comment]
1) I, for one, truly hope you get the grant.
2) connected to website theme, though lower priority I suspect, is continuing upon the groundwork which was done to make it straight-forward for a public organization to attach their own forward to provide context for within their organization

**Comment by clausmullie** [comment]
1) Wonderful initiative, happy to support if/where I can
2) Yes, on the top of my head, I'd say first white labeling the theme ([notes here](https://github.com/standard-for-public-code/standard-for-public-code/discussions/1037)), then enabling translations, then enabling organizations to have their own branding on their fork

Another thing is that there now seem to be 2 conflicting calendar invites which are managed by the Foundation for Public Code. I suspect a sync error somewhere. It would be nice to setup a fresh one in an environment several maintainers have access rights to. 

A quick search brought me to
* https://calendar.online/ perhaps - simple access link, but no email reminders or password protection in free version
* https://ical.marudot.com/ - making an iCal file, repeat and reminders incl, but no invite list or ability to change later on
Anyone have alternative suggestions?

**Comment by zorp** [comment]
1. 100% support from me. Would be fantastic.

2. I think your suggestions are excellent and @clausmullie have a point in looking at a cleanup of the calendar invites and figuring out a way to make it more accessible.. I myself forget the open calls because they are no longer in my calendar.

**Comment by Ainali** [comment]
I have submitted an application aligned with all the ideas above. Based on the timing in the previous round, the outcome will likely come by the end of December. :crossed_fingers:

**Comment by jgroenen** [comment]
For the Dutch translation I removed some stuff https://github.com/codefornl/standard-for-public-code-jekyll-theme/commits/master/
I think it would make sense to combine this with re-achitecturing the "community translations" setup: https://github.com/standard-for-public-code/community-translations-standard/discussions/45

**Comment by Ainali** [comment]
The grant was denied. The main feedback was that the application wasn't clear enough about what value the deliverables would have.

**Reply by ericherman** [reply]
That is good feedback.

**Reply by zorp** [reply]
On to the next one. Time is working for us.
